cc-db-service/openapi.yaml

```diff
@@ -0,0 +1,438 @@
+openapi: 3.0.3
+info:
+  version: 1.0.0
+  title: CC Database service endpoints
+
+security:
+  - bearerAuth: []
+
+paths:
+  /v1/system/ping:
+    get:
+      operationId: ping
+      summary: ping - pong
+      description: pong
+      tags:
+        - system
+      responses:
+        '200':
+          description: Success
+        '404':
+          description: Not found
+        '500':
+          description: Service error
+
+  /v1/apis:
+    get:
+      operationId: getApis
+      summary: Get API configurations
+      tags:
+        - Apis
+      responses:
+        '200':
+          description: List Apis
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/GetApi'
+        '404':
+          description: Not found
+        '500':
+          description: Service error
+  /v1/user:
+    post:
+      operationId: postUser
+      summary: Create new or return existing User data from database
+      tags:
+        - User
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PostUser'
+        required: true
+      responses:
+        '200':
+          description: Success
+        '400':
+          description: Invalid data
+        '404':
+          description: Not found
+        '500':
+          description: Service error
+  /v1/items/{userId}:
+    get:
+      operationId: getUserItems
+      description: Get Items belonging to a User
+      tags:
+        - Items
+      parameters:
+        - name: UserId
```

Comment:
Remember to write a parameter name that appears in the path the same way that it is written in the path (upper and lower case letters). So change this name from 'UserId' to 'userId'.